pkgs/tools/networking/zrok-bin/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, lib, fetchzip, patchelf }:
+
+stdenv.mkDerivation rec {
+  pname = "zrok-bin";
+  version = "0.3.4";
+
+  meta = {
```

Comment:
meta should be last according to the contributing guide.

Reply:
I checked both https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md and the manual https://nixos.org/nixpkgs/manual/#chap-submitting-changes and I don't find that requirement.
I saw it in another package and also thought it was odd.
In any case I moved it down.

Reply:
It should be in https://github.com/NixOS/nixpkgs/tree/master/doc/contributing